src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Windows/MyExtensionsApp.Windows.csproj

```diff
@@ -15,7 +15,6 @@
 		<!-- <WindowsAppSDKSelfContained>true</WindowsAppSDKSelfContained> -->
 		<!-- This bundles the .NET Core libraries (Uncomment for packaged builds)  -->
 		<SelfContained>true</SelfContained>
-		<DefaultLanguage>en</DefaultLanguage>
```

Comment:
Why are we removing this?

Reply:
We already have this defined in the Directory.Build.props so it isn't needed in the Windows head